packages/terra-functional-testing/src/commands/validates/element.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-unused-vars */
+const accessibility = require('./accessibility');
+
+/**
+ * An assertion method to assert the page is accessible and the screenshot comparison result is within
+ * the mismatch tolerance.
+ *
+ * This should be used within a Mocha `it` block.
+ *
+ * @param {Object} [options] - the test options
+ * @param {string} [options.testName] - the test case name.
+ * @param {Object} [options.rules] - the axe rules to use to assert accessibility.
+ * @param {Number} [options.misMatchTolerance] - the mismatch tolerance for the screenshot comparison.
+ * @param {string} [options.selector] - the element selector to use for the screenshot comparison.
+ */
+const element = (options = {}) => {
+  const { rules, misMatchTolerance } = options;
+
+  const selector = options.selector || global.Terra.serviceOptions.selector;
+  const testName = options.testName || 'default';
```

Comment:
nit: I think we should comment this code out until we need it (or remove it) so we don't have to disable the unused vars at the top of the file. It can be added back/uncommented when we integrate the screenshot logic

Reply:
Agree: https://github.com/cerner/terra-toolkit/pull/464/commits/2f33038f4383a4ecf1cfc25ecdd75d06d47d5469